src/CNAME

```diff
@@ -1 +1 @@
-wb.map.sensors.africa 
\ No newline at end of file
+//wb.map.sensors.africa 
```

Comment:
Why are we commenting this out?

Reply:
@kilemensi Sorry, forgot to uncomment it. The map is not showing with the CNAME link in it on local.